drivers/spi/spi_ll_stm32.c

```diff
@@ -1160,6 +1165,12 @@ static const struct spi_stm32_config spi_stm32_cfg_##id = {		\
 	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32_spi_subghz),	\
 		(.use_subghzspi_nss =					\
 			DT_INST_PROP_OR(id, use_subghzspi_nss, false),))\
+	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),	\
+		(.midi_clocks =					\
+			DT_INST_PROP_OR(id, midi_clock, 0),))\
+	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),	\
+		(.mssi_clocks =					\
+			DT_INST_PROP_OR(id, mssi_clock, 0),))\
```

Comment:
Please mind indentation

```suggestion
	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),		\
		(.midi_clocks =						\
			DT_INST_PROP_OR(id, midi_clock, 0),))		\
	IF_ENABLED(DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi),		\
		(.mssi_clocks =						\
			DT_INST_PROP_OR(id, mssi_clock, 0),))		\
```

Reply:
my tab size value did keep as 4, i did update and solve. :)